src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -156,7 +156,7 @@
             :aria-label="`Copy (${ALT_SHORTCUT_LABEL}+C)`"
             class="code-block-copy-button"
             :copy-tooltip="`Copy (${ALT_SHORTCUT_LABEL}+C)`"
-            data-testid="code-block-copy-button"
+            :data-testid="`code-block-copy-button${id ? '-' + id : '' }`"
```

Comment:
if id is a required prop, how would it be empty...?

Reply:
`''` is considered falsey for prop `{ type: String, required: true }`

Reply:
yeah, i'll just remove the logic